test/test_aptly_command.rb

```diff
@@ -39,14 +39,12 @@ def password(_prompt)
     options.port = 9000
     options.username = 'me'
     options.password = 'secret'
-    options.debug = true
```

Comment:
Need to revisit this.  If we have this set the `rake test` output will include the debugging output (pretty noisy)